docs/clustering.md

```diff
@@ -72,14 +72,23 @@ Recommendations and constraints:
 
 ### Provisioning and Deprovisioning Instances and Groups
 
-* **Provisioning** - Provisioning Instances after installation is supported by updating the `inventory` file and re-running the setup playbook. It's important that this file contain all passwords and information used when installing the cluster, or other instances may be reconfigured (this can be done intentionally).
+* **Provisioning** - Provisioning instances after installation is supported by updating the `inventory` file and re-running the setup playbook. It's important that this file contains all passwords and related information used when installing the cluster; if this is not the case, other instances may be reconfigured (this can be done intentionally).
 
-* **Deprovisioning** - AWX does not automatically de-provision instances since it cannot distinguish between an instance that was taken offline intentionally or due to failure. Instead, the procedure for de-provisioning an instance is to shut it down (or stop the `automation-controller-service`) and run the AWX de-provision command:
+* **Deprovisioning** - Prior to version version 19.3.0, AWX did not automatically deprovision instances since it could not distinguish between an instance that was taken offline intentionally or due to failure. Instead, the procedure for deprovisioning an instance was to shut it down (or stop the `automation-controller-service`) and run the AWX deprovision command:
```

Comment:
Even in version 19.3.0 and after, we still don't automatically deprovision offline nodes do we? Running under the awx-operator, I think some version of this does happen, but otherwise it doesn't happen for normal cluster nodes or execution nodes IIRC